index.html

```diff
@@ -4,9 +4,6 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  
```

Comment:
- [ ] `required`
- [ ] Amazing job so far 👏 
- [ ] the text is overlapping please see to it that it is staying the boxes a the origin design
- [ ] also the `join conference button` is not responsive please make reponsive

- [ ] `your template`
![image](https://user-images.githubusercontent.com/66230306/198561678-a6603926-1dfb-4193-a97b-a19d9698ead3.png)
![image](https://user-images.githubusercontent.com/66230306/198563082-ac3804ad-50aa-448a-9fb5-28f2d5994405.png)



`origin design`
![mainspeaker](https://user-images.githubusercontent.com/66230306/198561232-a52eabf8-cefd-488a-8ddc-17b750c64a33.png)